provider/testdata/Cargo.toml

```diff
@@ -66,7 +66,9 @@ locales = [
 
     # French:
     # - Often the first non-English locale to receive new data in CLDR
+    # - Belgian region has non default weekData.
     "fr",
+    "fr-BE",
```

Comment:
`concern`: I'm concerned about growing the test locales for every feature we want to measure. I don't think this feature justifies adding another locale.

Reply:
I've removed fr-BE and switched test cases to en-ZA (which has a first_weekday of Sunday) & removed cases that can no longer be tested (y can no longer be distinguished from Y given that all test locales use min_week_days = 1).